From cc72f9b1dfc33f5f984bddc2cfe417fd2fd3c053 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 16 Jul 2024 11:59:06 +0200
Subject: [PATCH 1199/1695] dts: stm32: remove support of st,clksrc on
 stm32mp25 boards

Remove unused "st,clksrc = <0>" in OPP configuration as PLL1 is always
used as CPU clock source.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git afb19bf910252b311189738cdc9ec106c381c5e2 dts: stm32: remove support of st,clksrc on stm32mp25 boards]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I9aaf7917eb028c05b66a4e5c8d8078a832c9ce10
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/393485
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts  | 2 --
 core/arch/arm/dts/stm32mp257f-ev1.dts | 2 --
 2 files changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 3c6b3a11f..9397e733c 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -390,13 +390,11 @@
 		st,ck_cpu1 {
 			cfg_1 {
 				hz = <1500000000>;
-				st,clksrc = <0>;
 				st,pll = <&pll1_cfg_1500Mhz>;
 			};
 
 			cfg_2 {
 				hz = <1200000000>;
-				st,clksrc = <0>;
 				st,pll = <&pll1_cfg_1200Mhz>;
 			};
 		};
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index dbcafe138..45ea74d5b 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -418,13 +418,11 @@
 		st,ck_cpu1 {
 			cfg_1 {
 				hz = <1500000000>;
-				st,clksrc = <0>;
 				st,pll = <&pll1_cfg_1500Mhz>;
 			};
 
 			cfg_2 {
 				hz = <1200000000>;
-				st,clksrc = <0>;
 				st,pll = <&pll1_cfg_1200Mhz>;
 			};
 		};
-- 
2.47.2

